dashboard/src/components/ChartView/ChartDeployButton.test.tsx

```diff
@@ -28,17 +28,30 @@ it("renders a button to deploy the chart version", () => {
   expect(button.text()).toBe("Deploy using Helm");
 });
 
-it("renders a redirect when the button is clicked", () => {
-  const wrapper = shallow(<ChartDeployButton version={testChartVersion} namespace="test" />);
-  const button = wrapper.find("button");
-  expect(button.exists()).toBe(true);
-  expect(wrapper.find(Redirect).exists()).toBe(false);
+it("renders a redirect with the correct URL when the button is clicked", () => {
```

Comment:
maybe we can revert this change since we don't have the special case for the _all namespace?

Reply:
I think the test is fine. It's not related to the issue but since it's already written it's good to have it.

Reply:
I actually like it because now it makes clear that the route depends on the namespace and the chart version.